response/slack/client.py

```diff
@@ -43,6 +43,19 @@ def get_user_id(self, name):
                 return user["id"]
         raise SlackError(f"User '{name}' not found")
 
+    def get_channel_name(self, id):
+        logger.info(f"Getting channel name for {id}")
```

Comment:
This is likely to be quite noisy

Reply:
> LGTM apart from the log line, also need to consider slack API limits though.

`conversations.info` is [Tier 3](https://api.slack.com/docs/rate-limits#tier_t3), meaning a rate limit of 50+ per minute, with "Sporadic bursts are welcome". I guess we could easily hit this with multiple concurrent users hitting the `/api/incidents` response, but I think we could easily introduce some rudimentary caching if needed (even caching the channel name for a minute or two in memory would probably fix this). I'd rather do this in a future PR though